packages/react-reconciler/src/ReactFiberRoot.js

```diff
@@ -106,7 +125,24 @@ export function createFiberRoot(
     expirationTime: NoWork,
     firstBatch: null,
     nextScheduledRoot: null,
+
+    nextGlobalRoot: null,
+    previousGlobalRoot: lastGlobalRoot,
+    setContext: (setRootContext.bind(null, uninitializedFiber): any),
   };
   uninitializedFiber.stateNode = root;
+  uninitializedFiber.memoizedState = {
+    element: null,
+    contexts: new Map(),
```

Comment:
I'm not sure we need this Map when the data is already in the update queue but do we need to eagerly initialize this whole thing?